addons/iot_box_image/overwrite_before_init/etc/init_image.sh

```diff
@@ -221,6 +221,7 @@ groupadd usbusers
 usermod -a -G usbusers odoo
 usermod -a -G lp odoo
 usermod -a -G input lightdm
+usermod -a -G pi odoo
```

Comment:
allow `odoo` access `/etc/hostapd/hostapd.conf` as it belongs to `pi` user and group.
Will also be a safety for `odoo` user to be able to access all `pi` files